lib/MusicBrainz/Server/Entity/EditNote.pm

```diff
@@ -125,7 +126,7 @@ around TO_JSON => sub {
 
     my $json = $self->$orig;
     $json->{editor_id} = $self->editor_id + 0;
-    $json->{editor} = $self->editor->TO_JSON;
+    $json->{editor} = to_json_object($self->editor);
```

Comment:
I guess we should change `EditNoteT` to use `+editor: EditorT | null,` then (it'll be loaded in most cases, but).

Reply:
Alternatively, we could load the editor here, but I guess it's not particularly useful just for consistency and having a check if needed on the React side is better.

Reply:
If the editor doesn't need to be loaded there I guess I'd prefer changing the type and seeing how much Flow likes that -- but if it causes a million errors we could always-load if it's more convenient.

Reply:
Didn't seem too problematic, made that change :)